packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -81,7 +71,7 @@ export default function ShareModal({
           columns={columns}
           column={column}
           row={row}
-          onSquadShare={() => onRequestClose(null)}
+          onClose={() => onRequestClose(null)}
```

Comment:
To be more accurate and avoid confusion since there is already an `onSquadsShare` prop inside.